project_task_create_portal/README.rst

```diff
@@ -0,0 +1,124 @@
+============================
+Project Portal Task Creation
+============================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:f6fee54148af1a81d39ccb70dc0f942f5a66150694fcebf82c733be3177601b0
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproject-lightgray.png?logo=github
+    :target: https://github.com/OCA/project/tree/16.0/project_task_create_portal
+    :alt: OCA/project
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/project-16-0/project-16-0-project_task_create_portal
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/project&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module extends the Project application by enabling portal users to create new tasks and edit their own tasks directly from the Odoo portal.
+
+
+## Features
+
+- New field on project: **Portal Task Creation Stage** 
+- Portal users can create tasks in the pre-defined stages for allowed projects
+- Extended portal UI: 
+  - **“Create Task”** button visible for allowed project
+  - Simplified task creation form (Name, Description, Deadline)  
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+## To configure the project for task creation from the portal
+
+- Go to the Project App
+- On a project kanban card, press the Settings  button
+- Go to the "Settings" tab
+
+- Set the **Portal Task Creation Stage** to a stage (from the project’s stages) from which portal users will be allowed to add tasks. 
+- Se the **Hide assigned user** to hide the assigned user of the task on the portal.
+
+Usage
+=====
+
+## To create a task from the portal
+
+
+- Go to the portal, navigate to Projects
+- Select an available project 
+- Press the “New Task” button
+- In the form view, fill in:
+   - Name (required)  
+   - Description (required)    
+   - Deadline (optional) 
+- Press the Save button
+
+## To edit a task from the portal
+
+Note: the user can edit only their own tasks created
+
+- Go to the portal, navigate to Projects
+- Select an available project
+- Select a task in the stage permitted for task creation and task editing
```

Comment:
The configuration on project refers to only create, there seem to be no edit options.